storage/src/test/scala/io/delta/storage/integration/S3LogStoreUtilIntegrationTest.scala

```diff
@@ -0,0 +1,85 @@
+package io.delta.storage.integration
+
+import io.delta.storage.internal.{FileNameUtils, S3LogStoreUtil}
+import org.apache.hadoop.conf.Configuration
+import org.apache.hadoop.fs.Path
+import org.apache.hadoop.fs.s3a.S3AFileSystem
+import org.scalatest.Tag
+import org.scalatest.funsuite.AnyFunSuite
+
+import java.net.URI
+import scala.math.max
+import scala.math.ceil
+
+class S3LogStoreUtilIntegrationTest extends AnyFunSuite {
```

Comment:
scaladoc explaining how to execute this integration test

Reply:
Done in https://github.com/delta-io/delta/pull/1210/commits/68e7a04350a55bb825d7a84ae4d2072215def6e3